README.md

```diff
@@ -1,10 +1,11 @@
 # Mentoring new pandas contributors (team 2019)
 
-This is the list of the GitHub usernames of all the people interested in contributing to pandas:
+This is the list of the GitHub usernames of all the people interested in contributing to pandas: [Github Link](https://github.com/python-sprints/pandas-mentoring)
```

Comment:
Sorry I wasn't clear, what I meant was more adding a link to the existing `GitHub` work, like:

```suggestion
This is the list of the [GitHub](https://github.com/) usernames of all the people interested in contributing to pandas:
```

Do you mind having both changes (adding your username, and adding this link) in separate pull requests? It's not trivial, let me know if you need help.

Reply:
okay, I misunderstood you. I will change it

Reply:
No problem, my comment wasn't very clear, sorry about that. To change the link in this PR I think you should be able to do it directly from this page, with the option below "Suggested change". The tricky part is to have both changes in different PRs.